DEVELOPMENT.md

```diff
@@ -118,6 +118,22 @@ kubectl create clusterrolebinding cluster-admin-binding \
   --user="${USER}"
 ```
 
+### Install in custom namespace
+1. To install into a different namespace you will need to modify resources in the `./config`
+    - remove all `namespace: tekton` resources
+    - delete the `namespace.yaml
```

Comment:
Missing "`" at end of file name. Also can you link the config namespace file here?

Reply:
done